src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Figures {
```

Comment:
we don't need this `enum`, it'll be enough if you write figure name in `draw()` methods)